src/material/core/theming/_theming.scss

```diff
@@ -20,21 +20,23 @@ $_emitted-color: () !default;
 $_emitted-typography: () !default;
 $_emitted-density: () !default;
 
-// For a given hue in a palette, return the contrast color from the map of contrast palettes.
-// @param $palette
-// @param $hue
+/// For a given hue in a palette, return the contrast color from the map of contrast palettes.
+/// @param {Map} $palette The palette from which to extract a color.
+/// @param {String | Number} $hue The hue for which to get a contrast color.
+/// @returns {Color} The contrast color for the given palette and hue.
 @function get-contrast-color-from-palette($palette, $hue) {
   @return map.get(map.get($palette, contrast), $hue);
 }
 
 
-// Creates a map of hues to colors for a theme. This is used to define a theme palette in terms
-// of the Material Design hues.
-// @param $base-palette
-// @param $default
-// @param $lighter
-// @param $darker
-// @param $text
+/// Creates a map of hues to colors for a theme. This is used to define a theme palette in terms
+/// of the Material Design hues.
+/// @param {Map} $base-palette Map of hue keys to color values for the basis for this palette.
+/// @param {String | Number} $default The default hue for this palette.
+/// @param {String | Number} $lighter The "lighter" hue for this palette.
+/// @param {String | Number} $darker The "darker" hue for this palette.
+/// @param {String | Number} $text The "text" hue for this palette.
```

Comment:
nit: it might just be me, but it feels redundant to prefix every description with "The".